recipes/transformer-engine/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# From https://github.com/conda-forge/pytorch_sparse-feedstock/blob/main/recipe/build.sh
+
+set -euxo pipefail
+
+mkdir -p $PREFIX/etc/conda/activate.d
+cp $RECIPE_DIR/transformer-engine-env.sh $PREFIX/etc/conda/activate.d/
+
+if [[ ${cuda_compiler_version} != "None" && "$target_platform" == linux-64 ]]; then
+    export FORCE_CUDA="1"
+    if [[ ${cuda_compiler_version} == 12.6 ]]; then
+        export TORCH_CUDA_ARCH_LIST="5.0;6.0;6.1;7.0;7.5;8.0;8.6;8.9;9.0+PTX"
```

Comment:
And is this env-var used by TransformerEngine?  Or is this for `pytorch`?

Reply:
Probably for pytorch; I actually suspect we could just delete this